app/models/KafkaMessageReader.scala

```diff
@@ -0,0 +1,107 @@
+package models
+
+import java.io.{ByteArrayInputStream, InputStreamReader}
+import java.nio.ByteBuffer
+import java.util.zip.{GZIPInputStream, ZipException}
+
+import com.google.common.io.CharStreams
+import kafka.api.{FetchRequestBuilder, PartitionMetadata, TopicMetadataRequest}
+import kafka.consumer.SimpleConsumer
+import play.api.Play.current
+import play.api.{Logger, Play}
+
+/**
+  * Contains code to read messages from kafka.
+  */
+object KafkaMessageReader {
+
+  private val Kafkas = Play.configuration.getString("capillary.kafkas").getOrElse("127.0.0.1:9092")
```

Comment:
i'm going to leave this as-is for now because this is how the other main class in capillary works. `ZkKafka` (poorly named) follows a similar pattern. so if we want to clean this up at some point, we can do both together.